### ISSUE TITLE
Support for GNU Octave

### ISSUE THREAD

**Issue by sanjayankur31** [chlubba/catch22#12]
Hello,

Thanks for the great package! [We're working on including it in NeuroFedora now.](https://pagure.io/neuro-sig/NeuroFedora/issue/380) 

Is there any chance you could test the package with [GNU Octave](https://www.gnu.org/software/octave/) to support it? Octave is a Free/Open source Matlab compatible tool, so that will make catch22 even more accessible/Open. 

In the meantime, we'll support all the non Matlab bits in NeuroFedora. Thanks again.

**Comment by benfulcher** [comment]
The Matlab parts of _catch22_ are not required for the functionality of this repository in non-Matlab languages (they are included in this repository only to assist Matlab users).

**Comment by sanjayankur31** [comment]
Yes, we won't include the Matlab bits in the package for the moment. When we do have confirmation that they work with Octave, we'll add them in.
